nix/build-support/ld_library_path.nix

```diff
@@ -0,0 +1,10 @@
+{
+  pkgs,
+  lib,
+  stdenv,
+  enableX11 ? true,
+  enableWayland ? true,
+}:
+lib.makeLibraryPath (import ./build_inputs.nix {
```

Comment:
We're mixing `thisCase` and `this-case` and `this_case` all within the `nix/` folder. I understand `thisCase` because it matches Nix function style. We generally elsewhere in the project use hyphens.

Reply:
Nix files are generally in kebab-case